SnapshotTests/SurveyViewControllerDynamicTypeFontLandscapeTests.swift

```diff
@@ -0,0 +1,35 @@
+@testable import GliaWidgets
+import SnapshotTesting
+import XCTest
+
+final class SurveyViewControllerDynamicTypeFontLandscapeTests: SnapshotTestCase {
+    func test_emptySurvey_extra3Large() {
+        let viewController = Survey.ViewController(viewFactory: .mock(), environment: .init(notificationCenter: .mock), props: .emptyPropsMock())
```

Comment:
These lines are a bit long, perhaps they can be broken down.

Reply:
fixed [e7a8a2c](https://github.com/salemove/ios-sdk-widgets/pull/778/commits/e7a8a2c1b1547be59e3ded193dd4464c26467386)